dvc/tree/http.py

```diff
@@ -56,20 +55,16 @@ def __init__(self, repo, config):
         self.ssl_verify = config.get("ssl_verify", True)
         self.method = config.get("method", "POST")
 
-    def _auth_method(self, path_info=None):
+    def _auth_method(self):
         from requests.auth import HTTPBasicAuth, HTTPDigestAuth
 
-        if path_info is None:
-            path_info = self.path_info
```

Comment:
_auth_method was never called with explicit path_info, so consider this removing unused code :smile: